wp-parsely.php

```diff
@@ -136,6 +141,19 @@ public function initialize_settings() {
         add_settings_section('optional_settings', 'Optional Settings',
                              array($this, 'print_optional_settings'),
                              Parsely::MENU_SLUG);
+
+        $h = 'Your API secret is your secret code to access our API. It can be found at dash.parsely.com/yoursitedomain/settings/api .
+         (replace yoursitedown with your domain name, e.g. `mydomain.com`) If you don\'t access to our API, email your account manager .
```

Comment:
Maybe better to say "if you haven't purchased access to our API and would like to do so" ? I could see people thinking they're entitled to access automatically and getting confused.

Reply:
Does it make sense to link to the API documentation here? Don't want the settings UI to get too busy visually, but it could be useful context for someone encountring this for the first time.